lore/settings.py

```diff
@@ -169,6 +171,32 @@ def get_var(name, default):
     ('text/requirejs', 'requirejs.RequireJSCompiler'),
 )
 
+# Media and storage settings
+IMPORT_PATH_PREFIX = get_var('LORE_IMPORT_PATH_PREFIX', 'course_archives/')
+MEDIA_ROOT = get_var('MEDIA_ROOT', '/tmp/')
+LORE_USE_S3 = get_var('LORE_USE_S3', False)
+AWS_ACCESS_KEY_ID = get_var('AWS_ACCESS_KEY_ID', False)
+AWS_SECRET_ACCESS_KEY = get_var('AWS_SECRET_ACCESS_KEY', False)
+AWS_STORAGE_BUCKET_NAME = get_var('AWS_STORAGE_BUCKET_NAME', False)
+# Provide nice validation of the configuration
+if(
+        LORE_USE_S3 and
+        (not AWS_ACCESS_KEY_ID or
+         not AWS_SECRET_ACCESS_KEY or
+         not AWS_STORAGE_BUCKET_NAME)
+):
+    raise ImproperlyConfigured(
+        'You have enabled S3 support, but are missing either '
```

Comment:
Nitty Nit: Common English usage has "either" used with multiple choices, but for people who have nothing better to do with their time than argue about such things, "either" is a choice between two options.  Absolutely no one will care either way.